src/niscope/metadata/functions.py

```diff
@@ -1722,6 +1722,14 @@
             'note': '\nSome functionality, such as time stamping, is not supported in all\ndigitizers. Refer to `Features Supported by\nDevice <REPLACE_DRIVER_SPECIFIC_URL_1(features_supported_main)>`__ for\nmore information.\n'
         },
         'included_in_proto': True,
+        'method_templates': [ # TODO: This is currently hand-written. Update from generated export before submitting.
```

Comment:
Note: This will get resolved when I get a new export after the [hapigen PR](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/393606) goes in.

Reply:
Updated to pull this from d95 which includes the method_templates addition here.